src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.image_small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.field {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  line-height: 22px;
+  padding-left: 62px;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
```

Comment:
move it to the `html` tag

Reply:
while moving it to the html tag, instead of html rules, default input styles are applied and tests fail

<img width="944" alt="Screenshot" src="https://user-images.githubusercontent.com/109239608/188312978-59c10270-fa9f-4193-9769-a14499f1061c.png">